skrub/_gap_encoder.py

```diff
@@ -687,6 +690,12 @@ def partial_fit(self, X, y=None):
         )
         # Update self.H_dict_ with the learned encoded vectors (activations)
         self.H_dict_.update(zip(unq_X, unq_H))
+
+        # XXX: Maybe we should accumulate unq_V or the scaling factor somehow?
+        scaling_factor = _gap_scaling_factor(unq_V)
+        if not hasattr(self, "scaling_factor_") or scaling_factor != 0:
+            self.scaling_factor_ = scaling_factor
```

Comment:
```suggestion
        if not hasattr(self, "scaling_factor_"):
            scaling_factor = _gap_scaling_factor(unq_V)
            if scaling_factor != 0:
                self.scaling_factor_ = scaling_factor
```
Let us not compute the scaling factor if we are not using it